playbooks/robusta_playbooks/oom_killer.py

```diff
@@ -31,6 +31,65 @@ class OomKillerEnricherParams(ActionParams):
 NODE_MEMORY_THRESHOLD = 0.95
 
 
+@action
+def pod_oom_killer_enricher(
+    event: PodEvent,
+):
+    pod = event.get_pod()
+
+    resource_requests = pod_requests(pod)
+    resource_limits = pod_limits(pod)
+
+    finding = Finding(
+        title=f"Pod {pod.metadata.name} in namespace {pod.metadata.namespace} OOMKilled results",
+        aggregation_key="pod_oom_killer_enricher",
+        severity=FindingSeverity.HIGH
+    )
+    logs_blocks: List[BaseBlock] = []
+    oom_killed_status = None
+    for container_status in pod.status.containerStatuses:
```

Comment:
checkout get_oom_kills_from_pod
It does a lot of similar work

It has for example: get_oom_killed_state
You could probably use it

In addition, the code there takes only recent oom kills.
Can there be a situation that a container has old oomkills?

Reply:
yea, changing it to get most recent oomkill